File upnpglobalvars.h needs to be updated because it's still showing
version 1.3.1 instead of current 1.3.2


https://sourceforge.net/p/minidlna/bugs/353/


--- a/upnpglobalvars.h	2022-08-30 06:42:54.000000000 +0000
+++ b/upnpglobalvars.h	2023-01-17 10:39:00.413939650 +0000
@@ -57,7 +57,7 @@
 
 #include <sqlite3.h>
 
-#define MINIDLNA_VERSION "1.3.1"
+#define MINIDLNA_VERSION "1.3.2"
 
 #ifdef NETGEAR
 # define SERVER_NAME "ReadyDLNA"
